cmd/helm/lint.go

```diff
@@ -51,19 +51,33 @@ func init() {
 	RootCommand.AddCommand(lintCommand)
 }
 
-var errLintNoChart = errors.New("no chart found for linting (missing Chart.yaml)")
-var errLintFailed = errors.New("lint failed")
+var errLintNoChart = errors.New("No chart found for linting (missing Chart.yaml)")
```

Comment:
Hmm, you're right - looking it up and according to https://github.com/golang/go/wiki/CodeReviewComments#error-strings it should warn about it...
Shall I change it back?